skaffold/src/test/kotlin/com/google/container/tools/skaffold/SkaffoldFilesTest.kt

```diff
@@ -35,6 +36,14 @@ class SkaffoldFilesTest {
         assertThat(isSkaffoldFile(skaffoldFile)).isTrue()
     }
 
+    @Test
+    fun `valid skaffold file not named skaffold_yaml is accepted`() {
```

Comment:
did you mean "not named skaffold**.**yaml"? (accidental underscore?)

Reply:
it doesn't allow dots in this type of names, so I used an underscore.

Reply:
oh I see. maybe something like skaffold[dot]yaml ? not a big deal

Reply:
square brackets not allowed either :) good to know for us for later tests names.